addons/mail/static/src/composer/composer.xml

```diff
@@ -63,6 +64,7 @@
                         />
                     </div>
                     <div class="o-mail-Composer-actions d-flex bg-view"
+                        t-if="!thread or thread.type !== 'chatter' or thread.hasWriteAccess or (thread.hasReadAccess and thread.canPostOnReadonly)"
```

Comment:
You should not disable all actions when thread is read-only. Some actions don't necessarily require write access.
You should add this to each condition of actions, one by one, so that some actions could be eligible on read-only threads. (e.g. open chat window conversation in discuss or search, they don't necessarily require write access)

Reply:
If the user is not able to send any messages or attachments then what is the need of showing the actions which are related to them? `open chat window conversation in discuss` and `search` are not part of composer actions as per my knowledge.

do let me know if i am missing something.